pkgs/development/python-modules/pgvector/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, numpy
+, pythonOlder
+}:
+
+buildPythonPackage rec {
+  pname = "pgvector";
+  version = "0.1.8";
+  format = "setuptools";
+
+  disabled = pythonOlder "3.6";
+
+  src = fetchFromGitHub {
+    owner = "pgvector";
+    repo = "pgvector-python";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-+0daPPZGVkqfZw0gOZwIl+knL/zZki9fs5kA3dYqPpE=";
+  };
+
+  propagatedBuildInputs = [
+    numpy
+  ];
+
+  # tests require db access
+  doCheck = false;
```

Comment:
We have `postgresqlTestHook` to set up a postgresql database during tests.

Reply:
Thanks for the suggestion.
Since I needed to set `postgresqlEnableTCP=1`, the checkPhase only ran on Linux.